sea_itt_lib/Utils.cpp

```diff
@@ -120,22 +123,25 @@ std::string GetStackString()
 #ifdef _WIN32
     return std::string();
 #else
-    // TStack stack = {};
-    // size_t size = GetStack(stack);
-    //
-    // char **bt_syms = backtrace_symbols(stack, size);
-    // if (!bt_syms)
-    //     return std::string();
-    // std::string res;
-    // for (int i = 2; i < size; i++) {
-    //     if (res.size())
-    //         res += "<-";
-    //     res += bt_syms[i];
-    // }
-    //
-    // free(bt_syms);
-    // return res;
-    return std::string();
+    #ifdef USE_MUSL_LIBC
+        return std::string();
+    #else
+        TStack stack = {};
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be executed?

Reply:
If using MUSL's libc, none of the backtrace code works so @raphaelcohn commented it out; now a default string will be returned if MUSL libc is used or the pre-existing code will be run otherwise.